entity_embed/benchmarks/base.py

```diff
@@ -138,20 +127,5 @@ def _read_pair_sets(self, pair_csv_path):
 
         return pos_pair_set, neg_pair_set
 
-    def build_datamodule(self, record_numericalizer, batch_size, eval_batch_size, random_seed):
```

Comment:
Removed this, because we're not making benchmarks with our classes. It's not "fair", as we operate on clusters, not pairs.